# Translations template for neutron.
# Copyright (C) 2014 ORGANIZATION
# This file is distributed under the same license as the neutron project.
#
# Translators:
msgid ""
msgstr ""
"Project-Id-Version: Neutron\n"
"Report-Msgid-Bugs-To: EMAIL@ADDRESS\n"
"POT-Creation-Date: 2014-09-14 06:11+0000\n"
"PO-Revision-Date: 2014-07-18 21:01+0000\n"
"Last-Translator: openstackjenkins <jenkins@openstack.org>\n"
"Language-Team: Portuguese (Brazil) (http://www.transifex.com/projects/p/"
"neutron/language/pt_BR/)\n"
"Language: pt_BR\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Generated-By: Babel 1.3\n"
"Plural-Forms: nplurals=2; plural=(n > 1);\n"

#: neutron/policy.py:94
#, python-format
msgid ""
"Found deprecated policy rule:%s. Please consider upgrading your policy "
"configuration file"
msgstr ""
"Encontrada regra deprecada: %s. Por favor considere atualizar seu arquivo de "
"regras de políticas"

#: neutron/policy.py:137
#, python-format
msgid "Unable to find data type descriptor for attribute %s"
msgstr ""
"Não foi possível encontrar o descritor de tipo de dados para o atributo %s"

#: neutron/agent/securitygroups_rpc.py:183
msgid ""
"security_group_info_for_devices rpc call not supported by the server, "
"falling back to old security_group_rules_for_devices which scales worse."
msgstr ""

#: neutron/agent/linux/keepalived.py:358
msgid ""
"A previous instance of keepalived seems to be dead, unable to restart it, a "
"new instance will be spawned"
msgstr ""

#: neutron/db/l3_agentschedulers_db.py:105
msgid ""
"Time since last L3 agent reschedule check has exceeded the interval between "
"checks. Waiting before check to allow agents to send a heartbeat in case "
"there was a clock adjustment."
msgstr ""

#: neutron/db/l3_agentschedulers_db.py:127
#, python-format
msgid ""
"Rescheduling router %(router)s from agent %(agent)s because the agent did "
"not report to the server in the last %(dead_time)s seconds."
msgstr ""

#: neutron/db/l3_hamode_db.py:404
#, python-format
msgid "The router %(router_id)s is bound multiple times on the agent %(host)s"
msgstr ""

#: neutron/openstack/common/service.py:364
#, python-format
msgid "pid %d not in child list"
msgstr "pid %d fora da lista de filhos"

#: neutron/plugins/ml2/drivers/type_tunnel.py:143
#, python-format
msgid "%(type)s tunnel %(id)s not found"
msgstr ""

#: neutron/plugins/ml2/drivers/cisco/apic/apic_sync.py:68
#, python-format
msgid "Create network postcommit failed for network %s"
msgstr ""

#: neutron/plugins/ml2/drivers/cisco/apic/apic_sync.py:78
#, python-format
msgid "Create subnet postcommit failed for subnet %s"
msgstr ""

#: neutron/plugins/ml2/drivers/cisco/apic/apic_sync.py:91
#, python-format
msgid "Create port postcommit failed for port %s"
msgstr ""

#: neutron/plugins/ml2/drivers/cisco/apic/apic_sync.py:110
#, python-format
msgid "Add interface postcommit failed for port %s"
msgstr ""

#: neutron/plugins/nuage/syncmanager.py:299
#, python-format
msgid "Subnet %s not found in neutron for sync"
msgstr ""

#: neutron/plugins/nuage/syncmanager.py:317
#, python-format
msgid "Router %s not found in neutron for sync"
msgstr ""

#: neutron/plugins/nuage/syncmanager.py:329
#, python-format
msgid ""
"Route with destination %(dest)s and nexthop %(hop)s not found in neutron for "
"sync"
msgstr ""

#: neutron/plugins/nuage/syncmanager.py:343
#, python-format
msgid "Security group %s not found in neutron for sync"
msgstr ""

#: neutron/plugins/nuage/syncmanager.py:354
#, python-format
msgid "Security group rule %s not found in neutron for sync"
msgstr ""

#: neutron/plugins/nuage/syncmanager.py:364
#, python-format
msgid "Floating ip %s not found in neutron for sync"
msgstr ""

#: neutron/plugins/nuage/syncmanager.py:376
#, python-format
msgid "IP allocation for floating ip %s not found in neutron for sync"
msgstr ""

#: neutron/plugins/nuage/syncmanager.py:387
#, python-format
msgid "Net-partition %s not found in neutron for sync"
msgstr ""

#: neutron/plugins/nuage/syncmanager.py:397
#, python-format
msgid "VM port %s not found in neutron for sync"
msgstr ""

#: neutron/plugins/ofagent/agent/ofa_neutron_agent.py:290
#, python-format
msgid "Unable to create tunnel port. Invalid remote IP: %s"
msgstr ""

#: neutron/plugins/ofagent/agent/ofa_neutron_agent.py:713
#, python-format
msgid ""
"VIF port: %s has no ofport configured, and might not be able to transmit"
msgstr ""

#: neutron/plugins/ofagent/agent/ofa_neutron_agent.py:820
#, python-format
msgid "Device %s not defined on plugin"
msgstr ""
